redfish-core/lib/log_services.hpp

```diff
@@ -1413,6 +1462,34 @@ inline void requestRoutesJournalEventLogEntry(App& app)
             });
 }
 
+static bool isHiddenEventLogEntry(std::string& path)
+{
+    constexpr auto loggingService = "xyz.openbmc_project.Logging";
```

Comment:
You can't combine this with the ?

                "xyz.openbmc_project.Logging", "/xyz/openbmc_project/logging",
                "org.freedesktop.DBus.ObjectManager", "GetManagedObjects");

Reply:
It is possible to combine, but doing so will lead to walk through all properties corresponding to  xyz.openbm_project.Logging.Entry - which is not necessary when the Hidden property is true. Also, there are other properties under org.open_power.Logging.PEL.Entry interface. We don't need to look through those props as well. Having a separate function specifically to check only the Hidden property is cleaner in my opinion.  But it is desired to use GetManagedObject, I can make that change.

Reply:
It is desired to only have 1 dbus call